docs/elements.md

```diff
@@ -315,15 +315,15 @@ def fib(n):
 | `min-lines`          | integer | —                    | Minimum number of lines the editor should show initially.                                                                                                                                                                                                                                                                                                                                                                                                 |
 | `max-lines`          | integer | —                    | Maximum number of lines the editor should display at once. Must be greater than `min-lines`.                                                                                                                                                                                                                                                                                                                                                              |
 | `auto-resize`        | boolean | true                 | Automatically expand the editor panel to ensure all lines are present. Overrides any value set by `"max-lines"` and establishes a default of 18 lines for `"min-lines"` if not supplied. See Details below for notes.                                                                                                                                                                                                                                     |
-| `preview`            | string  | —                    | If set, provides a live preview mode for editing markup languages. Currently supports `"html"`, `"markdown"`, or `"dot"`, though additional languages are supported via extensions (see below).                                                                                                                                                                                                                                                           |
+| `preview`            | string  | —                    | If set, provides a live preview mode for editing markup languages. Currently, `"html"`, `"markdown"`, or `"dot"` are supported, though additional languages are supported via extensions (see below).                                                                                                                                                                                                                                                     |
```

Comment:
Nit: "supported" sounds repetitive now. Maybe replace the first one with "available"?

Reply:
Changed in [f2429b4](https://github.com/PrairieLearn/PrairieLearn/pull/11643/commits/f2429b43439d5e25b6eb82f254e7d55f94d69384).